modules/metastore/metastore.module

```diff
@@ -114,9 +114,10 @@ function metastore_form_alter(&$form, \Drupal\Core\Form\FormStateInterface $form
     if (empty($json)) {
       return;
     }
-    $object = json_decode($json);
+    // TODO: use an actual schema id.
```

Comment:
I don't have the schema id at this stage.

In my latest commits I used `field_data_type` for the schema id here. However, I left a similar todo in the hook_fiorm_alter() above because I'm not sure if I have the entity object there - I couldn't pause it with a breakpoint to debug

Reply:
gotcha, ok